tests/integration/backward_compatible/sql_test.py

```diff
@@ -178,20 +217,20 @@ def test_execute_with_params(self):
     def test_execute_with_mismatched_params_when_sql_has_more(self):
         self._create_mapping()
         self._populate_map()
-        result = self.client.sql.execute(
-            'SELECT * FROM "%s" WHERE __key > ? AND this > ?' % self.map_name, 5
-        )
 
         with self.assertRaises(HazelcastSqlError):
+            result = self.execute(
+                'SELECT * FROM "%s" WHERE __key > ? AND this > ?' % self.map_name, 5
+            )
             for _ in result:
                 pass
```

Comment:
```suggestion
        with self.assertRaises(HazelcastSqlError):
            self.execute(
                'SELECT * FROM "%s" WHERE __key > ? AND this > ?' % self.map_name, 5
            )
```

Reply:
Can't do that, for backward compatibility tests as 4.x clients throw while using SQL result